gnucash/import-export/ofx/gnc-ofx-import.cpp

```diff
@@ -1475,7 +1477,7 @@ void gnc_file_ofx_import (GtkWindow *parent)
 
         DEBUG("Opening selected file(s)");
         // Create the structure that holds the list of files to process and the statement info.
-        info = g_new(ofx_info,1);
+        info = new ofx_info;
```

Comment:
If it's the info getting deleted at 1199 then it's safe.

Reply:
It's not upgraded. There's no difference in the returned pointer, it just comes from a different memory pool. 

> because info will be sent via ofx_set_statement_cb(libofx_context, ofx_proc_statement_cb, info)

Not exactly. `ofx_set_statement_cb` tells libofx that [`ofx_proc_statement_cb`](https://github.com/Gnucash/gnucash/blob/0a9e999cfd9f0f4c19cf8c1b79c9c8c1285af542/gnucash/import-export/ofx/gnc-ofx-import.cpp#L1016) is our statement-processing callback and that libofx should pass `info` as the second argument to `ofx_proc_statement_cb`.  [`struct _ofx_info`](https://github.com/Gnucash/gnucash/blob/0a9e999cfd9f0f4c19cf8c1b79c9c8c1285af542/gnucash/import-export/ofx/gnc-ofx-import.cpp#L72) contains GnuCash variables that libofx doesn't know anything about.

Reply:
Ok. Last commit does upgrade info.